<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDlg</class>
 <widget class="QMainWindow" name="MainDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainDlg</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DGSW_project</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSignButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>400</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStartButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>400</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStopButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>400</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="drivName">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="drivCarNum">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="drivCarMod">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
     </font>
    </property>
    <property name="text">
     <string>Car Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
     </font>
    </property>
    <property name="text">
     <string>Car Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Driver's information</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1078</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
